auslib/web/common/swagger/definitions.yml

```diff
@@ -325,7 +325,7 @@ definitions:
         example: true
 
       jaws:
-        description: Whether or not the Rule should apply to queries that are incompatible with the JAWS screen reader. If set to false or true, the query and the rule must match precisely. If set to null, the jaws value in the updateQuery is ignored.
+        description: Whether or not the Rule should apply to queries that indicate an incompatible version of the JAWS screen reader. If set to false or true, the query and the rule must match precisely. If set to null, the jaws value in the updateQuery is ignored.
```

Comment:
I had to read through both linked bugs to understand this correctly. Could we maybe elaborate a little on this, either here or below, to say something like "queries that have JAWS installed but with a version before the fix that makes it compatible with e10s"? Also, where are the docs for arguments to the update function? We could add similar information there.

Reply:
Good idea - I'll improve on this. What do you mean when you say "update function"?

Reply:
I meant, where do we document the parameters to the /update URLs, and especially what `SYSTEM_CAPABILITIES` are valid and what they mean.

Reply:
Hmm, we don't actually have any update URL centric documentation that I can find (either in Balrog or elsewhere) - that's a very good idea though. For now, I'll do what I can to improve the clarity in the existing Balrog docs.